app/models/result.rb

```diff
@@ -12,4 +12,11 @@ def initialize(title, year, url, type)
     @url = url
     @type = type
   end
+
+  # Testing something
+  def truncate_authors(authors)
+    return @authors if !ENV['MAX_AUTHORS'] || authors.length <= ENV['MAX_AUTHORS'].to_i
```

Comment:
Negative conditions are sometimes hard to grok. Rails allows `unless` to allow you to not have to `!` things.

Also, I think we can just make this a required ENV and not do that check at all.